docs/CONTRIBUTING.md

```diff
@@ -1,25 +1,46 @@
 
 * Check to make sure it's not a duplicate of an existing issue.
 * Create a separate "issue" for each bug you are reporting and each feature you are requesting.
+* Choose the correct issue template and fill it out to the best of your ability.
 * Do not use the issues page for things other than bug reports and feature requests. Use the [discussions page](https://github.com/endless-sky/endless-sky/discussions) instead.
 
 If requesting a new feature, first ask yourself: will this make the game more fun or interesting? Remember that this is a game, not a simulator. Changes will not be made purely for the sake of realism, especially if they introduce needless complexity or aggravation.
 
-## Posting pull requests
+### Closing issues
+
+If you believe your issue has been resolved, you can close the issue yourself. If your issue gets closed because a PR was merged, and you are not satisfied, please open a new issue.
+
+## Pull requests
+
+### Posting pull requests
 
 If you are posting a pull request, please:
 
+* Find if your pull request solves an existing issue. Note that just because an issue is open does not guarantee that it is something we will accept, as the number of issues we receive means that not all of them have been thoroughly discussed, and there may be open issues for features that we decide we do not want.
 * Do not combine multiple unrelated changes.
 * Check the diff and make sure the pull request does not contain unintended changes.
 * If changing the C++ code, follow the [coding standard](https://endless-sky.github.io/styleguide/styleguide.xml).
+* Fill out the pull request template to the best of your ability. GitHub's pull request template features are not as thorough as issue templates. You're able to delete the entire pull request template and put in whatever you want; please do not do this. The purpose of the pull request template is to help facilitate the review, approval, and merging of pull requests.
 
 If proposing a major pull request, start by posting an issue and discussing the best way to implement it. Often the first strategy that occurs to you will not be the cleanest or most effective way to implement a new feature.
 
-## Closing issues
+### What to expect when opening a pull request
 
-If you believe your issue has been resolved, you can close the issue yourself. If your issue gets closed because a PR was merged, and you are not satisfied, please open a new issue.
+Endless Sky is developed by a community of volunteers. We're a collection of people with various different interests and skill sets, so the type of pull request you open will influence who will be looking at it.
+
+All pull requests go through a review process before being merged. We have a group of dedicated Reviewers who help provide reviews to pull requests as they come in. We also have a group of Developers who make determinations on what does and does not get merged, and are the ones who have the access to merge pull requests.
```

Comment:
>  who have the access to merge pull requests

Maybe permission would sound better? Or just say that they are the ones who merge PRs?

Reply:
```suggestion
All pull requests go through a review process before being merged. We have a group of dedicated Reviewers who help provide reviews to pull requests as they come in. We also have a group of Developers who make determinations on what does and does not get merged, and are the ones with the permissions to merge pull requests.
```